NovelRT.Sdk/TemplateFiles/conanfile.py

```diff
@@ -1,27 +1,19 @@
 from conans import ConanFile, CMake
-import os
 
 class ###PROJECT_NAME###Conan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake_find_package", "cmake_paths"
+    generators = "cmake_paths", "cmake_find_package"

```

Comment:
Why were these swapped around out of curiosity?

Reply:
Eh, they were modified and then put back..just in a different order

Reply:
Reversed in the newest commit, no longer an issue.